forch/faucet_state_collector.py

```diff
@@ -569,24 +623,35 @@ def _fill_path_to_root(self, switch_name, switch_map):
         """populate path to root for switch_state"""
         switch_map["root_path"] = self.get_switch_egress_path(switch_name)
 
-    def _fill_dva_states(self, switch_name, port_id, port_map):
-        dp_obj = self.faucet_config.get(DPS_CFG, {}).get(switch_name)
-        if not dp_obj:
+    def _fill_dva_states(self, switch_name, port_id, port_map, list_acls=False):
```

Comment:
at this point, there is no "d" (dynamic) -- it's simply filling out the states as they are assigned, correct?  i.e. if they were static vlan assignments, would it make any difference to this code?

Reply:
refactored